javascript-timers/interval.js

```diff
@@ -0,0 +1,13 @@
+let count = 4;
+
+const countdown = function startCountdown() {
+  const $countdownDisplay = document.querySelector('.countdown-display');
```

Comment:
This should be done globally. By including it inside the function, you're forcing us to traverse the DOM over and over every single time this function gets called